tests/packagedcode/test_npm.py

```diff
@@ -367,6 +367,21 @@ def test_npm_yarn_with_package_json_resolve_dependencies(self):
             expected_file, result_file, remove_uuid=True, regen=REGEN_TEST_FIXTURES
         )
 
+    @pytest.fixture
+    def handler():
+        return NpmPackageLockJsonHandler()
+
+    def test_npm_package_lock_json_parse(handler):
+        test_file = 'npm/package-lock-v1/package-lock.json'
+        expected_file = 'npm/package-lock-v1/package-lock.json-expected'
+        with open(test_file) as f:
+            sample_data = json.load(f)
+        handler.parse(test_file)
+        with open(expected_file) as f:
+            expected_data = json.load(f)
+        assert handler.packages == expected_data
```

Comment:
Please see how tests are written for parsing package manifests below, you need to follow the same structure. `check_packages_data` does all the checking for you, you just need to specify the input/output and parsing function.